addons/web_editor/models/ir_attachment.py

```diff
@@ -63,3 +63,12 @@ def _get_media_info(self):
         """Return a dict with the values that we need on the media dialog."""
         self.ensure_one()
         return self._read_format(['id', 'name', 'description', 'mimetype', 'checksum', 'url', 'type', 'res_id', 'res_model', 'public', 'access_token', 'image_src', 'image_width', 'image_height', 'original_id'])[0]
+
+    def _can_bypass_rights_on_media_dialog(self, **attachment_data):
+        """ This method is meant to be overridden, for instance to allow to
+        create image attachment despite the user not allowed to to create
```

Comment:
It can't really be, `_attachment_create` is inside a controller, not ideal to do inheritance.
In reality, `_attachment_create` should also be here, I wanted to move it also when introduced this method but I kept the minimal diff
Can still move it